packages/experiments/src/components/Toggle/Toggle.styles.ts

```diff
@@ -56,171 +55,152 @@ export const ToggleStyles: IToggleComponent['styles'] = props => {
+export const ToggleStyles: IToggleComponent['styles'] = (props, theme, tokens) => {
+  const { className, disabled, checked } = props;
+
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  const textStyles: IStyle = [
+    classNames.text,
+    {
+      selectors: {
+        // Workaround: make rules more sepecific than Label rules.
+        '&&': {
+          color: tokens.textColor,
+          padding: '0',
+          margin: '0 10px',
+          userSelect: 'none',
+          selectors: {
+            [HighContrastSelector]: {
+              color: tokens.textHighContrastColor
             }
           }
-        ],
-
-        container: [
-          classNames.container,
-          {
-            display: 'inline-flex',
-            position: 'relative'
+        }
+      }
+    }
+  ];
+
+  return {
+    root: [
+      classNames.root,
+      checked && 'is-checked',
+      !disabled && 'is-enabled',
+      disabled && 'is-disabled',
+      theme.fonts.medium,
+      {
+        marginBottom: '8px'
+      },
+      className
+    ],
+    label: [
+      classNames.label,
+      {
+        color: tokens.textColor,
+        selectors: {
+          [HighContrastSelector]: {
+            color: tokens.textHighContrastColor
           }
-        ],
-
-        pill: [
-          classNames.pill,
-          getFocusStyle(theme, -3),
-          {
-            fontSize: '20px',
-            boxSizing: 'border-box',
-            width: '2.2em',
-            height: '1em',
-            borderRadius: '1em',
-            transition: 'all 0.1s ease',
-            borderWidth: '1px',
-            borderStyle: 'solid',
-            background: state.pillBackground,
-            borderColor: state.pillBorderColor,
-            justifyContent: state.pillJustifyContent,
-            cursor: 'pointer',
-            display: 'flex',
-            alignItems: 'center',
-            padding: '0 .2em',
-            selectors: {
-              ':hover': [
-                {
-                  backgroundColor: state.pillHoveredBackground,
-                  borderColor: state.pillHoveredBorderColor,
-                  selectors: {
-                    [HighContrastSelector]: {
-                      backgroundColor: state.pillHighContrastHoveredBackground
-                    }
-                  }
-                }
-              ],
-              ':hover .ms-Toggle-thumb': [
-                {
-                  selectors: {
-                    [HighContrastSelector]: {
-                      borderColor: state.pillHighContrastHoveredBorderColor
-                    }
-                  }
+        }
+      }
+    ],
+    container: [
+      classNames.container,
+      {
+        display: 'inline-flex',
+        position: 'relative'
+      }
+    ],
+    pill: [
+      classNames.pill,
+      getFocusStyle(theme, -3),
+      {
+        fontSize: '20px',
+        boxSizing: 'border-box',
+        width: '2.2em',
+        height: '1em',
```

Comment:
i'm not sure why these EM values are here. Not a change for this pr but something maybe we can look into.